sources/packages/backend/apps/db-migrations/src/sql/NotificationMessages/Rollback-insert-parent-declaration-required-message.sql

```diff
@@ -0,0 +1,9 @@
+DELETE FROM
+  sims.notification_messages
+WHERE
+  ID = 32;
+
+DELETE FROM
+  sims.notification_messages
+WHERE
+  ID = 33;
```

Comment:
Can use a single delete with IN. I can see other notifications using that way to revert.